docker/Dockerfile.aarch64

```diff
@@ -0,0 +1,75 @@
+# author  : paulfantom
+# original: https://github.com/paulfantom/netdata
+# SPDX-License-Identifier: GPL-3.0+
+
+FROM resin/aarch64-debian:stretch as builder
+
+RUN [ "cross-build-start"]
+
+# Copy source
+COPY . /opt/netdata.git
+WORKDIR /opt/netdata.git
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install dependencies
+RUN apt-get -qq update && \
+    apt-get -y install zlib1g-dev uuid-dev libmnl-dev gcc make curl git autoconf autogen automake pkg-config netcat-openbsd jq autoconf-archive
+
+# Install from source
+RUN chmod +x netdata-installer.sh && \
+    sync && sleep 1 && \
+    ./netdata-installer.sh --dont-wait --dont-start-it
+
+# Copy entrypoint script
+RUN cp docker/run.sh /usr/sbin/run.sh && \
+    chmod +x /usr/sbin/run.sh
+
+RUN [ "cross-build-end"]
+
+#####################################################################
+FROM resin/aarch64-debian:stretch
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN [ "cross-build-start"]
+
+# Install runtime dependencies
+RUN apt-get -qq update && \
+    apt-get install --no-install-recommends -y python lm-sensors nodejs python python-mysqldb python-yaml jq curl fping netcat-openbsd && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
add
```
apt-get -y autoremove
apt-get clean
```

Reply:
What for? I don't have anything to remove or clean since `rm -rf /var/lib/apt/lists/*` cleans it.

Reply:
I'm not sure. Saw that in other jobs. Is there something in /var/cache/apt that is cleaned by this?

Reply:
This is based on kubernetes build system ([here](https://github.com/kubernetes-sigs/cluster-api/blob/a082a277bceaed15ae79f61b2ebba66201828ac4/cmd/apiserver/Dockerfile#L27)) and on hackernoon blog post ([here](https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34)). I have also checked what is the size of `/var/cache/apt` in a container:

```
root@ea5f76d07045:/var/cache/apt# du -hcd1
0       ./archives
0       .
0       total
```

So there is no need to add `apt-get clean`.

As for `apt-get -y autoremove`, there is no unnecessary artifacts so there is also no need to add that command. In some cases it can also cause more harm than good.